storage/2020-08-04/blob/blobs/README.md

```diff
@@ -30,7 +30,7 @@ func Example() error {
 
 	blobClient, err := blobs.NewWithBaseUri(fmt.Sprintf("https://%s.blob.%s", accountName, domainSuffix))
 	if err != nil {
-		t.Fatalf("building client for environment: %+v", err)
+		fmt.Errorf("building client for environment: %+v", err)
```

Comment:
We'll need to add `return` here too, since this method returns an error?

```suggestion
		return fmt.Errorf("building client for environment: %+v", err)
```